addons/account/views/account_move_views.xml

```diff
@@ -734,6 +734,7 @@
                         <field name="bank_partner_id" invisible="1"/>
                         <field name="display_qr_code" invisible="1"/>
                         <field name="show_reset_to_draft_button" invisible="1"/>
+                        <field name="amount_total"/>
```

Comment:
@william-andre considering the line just above, wouldn't it be better to create a boolean for that?

Directly related: would it be ok to create a computed non-stored field in stable?

Reply:
It is better to avoid it and I don't see a real added value in this case. A computed field would be for more complicated cases with complex inheritance that doesn't allow to change the attrs for instance